packages/components/src/DateTimePickers/InputDatePicker/InputDatePicker.component.js

```diff
@@ -60,13 +65,14 @@ export default function InputDatePicker(props) {
 		),
 	].filter(Boolean);
 	return (
-		<div className="date-picker">
+		<div className={classNames(theme['date-picker'], 'date-picker')}>
```

Comment:
To validate with UX, but if we never have the timezone info on date, you can remove this extra class